src/documentation/reference/ReferenceNode.tsx

```diff
@@ -77,9 +77,10 @@ const ReferenceNode = ({ anchor, docs, ...props }: ApiDocEntryNodeProps) => {
         fontSize="sm"
         spacing={3}
         p={5}
+        pb={3}
         pr={3}
         mt={1}
-        mb={kindToSpacing[kind]}
```

Comment:
I think the padding provides plenty of spacing and this only seems to add additional margin where we don't want it? Happy to be proven wrong.

Reply:
I agree, if anything we still have too much spacing between the methods/fields on a class:

![image](https://user-images.githubusercontent.com/44397098/151389588-21e262f9-f3e1-4caf-b028-982e53b37ec1.png)